cypress-tests/cypress/e2e/PaymentUtils/Cybersource.js

```diff
@@ -391,6 +404,78 @@ export const connectorDetails = {
         },
       },
     },
+    SaveCardUseNo3DSAutoCaptureOffSession: {
+      Request: {
+        payment_method: "card",
+        payment_method_data: {
+          card: successfulNo3DSCardDetails,
+        },
+        currency: "USD",
```

Comment:
Passing currency in confirm call is useless. You're expected to pass currency in Intent call.

Please remove it here and else where below